### ISSUE TITLE
Improve the security of the voting system

### ISSUE THREAD

**Issue by opticod** [NJACKWinterOfCode/E-voting-portal#3]
The current implemented voting protocol and system need to updated. Comment and consult your proposed ideas and then move on to creating PRs.

**Comment by vanshsoni** [comment]
Hey @opticod Can we store KeyId that is used for voting by encrypting it using MD5 algorithm as it not reversible algorithm
when user will vote then we will check using the same function to compare both KeyID encrypted msg
It will secure keyID leakage from database side

**Comment by opticod** [comment]
`MD5 algorithm as it not reversible algorithm`, give a MD5 encrypted text to me here, and I will decrypt it using tools like http://www.md5online.org/

**Comment by Logan1x** [comment]
I agree with @opticod  MD5 is so common nowadays that anyone can get decrypt it using online tools.

**Comment by vanshsoni** [comment]
ok
